scripts/livereduce.py

```diff
@@ -355,6 +377,11 @@ def process_default(self, event):
 # start up the live data
 liveDataMgr.start()
 
+# start the memory checker
+if config.system_mem_limit_perc > 0:
+    memory_thread = threading.Thread(target=memory_checker, args=(config, liveDataMgr))
```

Comment:
Adding `daemon=True` allowed me to stop livereduce with Ctrl+C
```suggestion
    memory_thread = threading.Thread(target=memory_checker, args=(config, liveDataMgr), daemon=True)
```